specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2021-03-01-preview/examples/ConfigurationStoresCreate.json

```diff
@@ -0,0 +1,89 @@
+{
+  "parameters": {
+    "subscriptionId": "c80fb759-c965-4c6a-9110-9b2b2d038882",
+    "resourceGroupName": "myResourceGroup",
+    "configStoreName": "contoso",
+    "api-version": "2021-03-01-preview",
+    "configStoreCreationParameters": {
+      "location": "westus",
+      "sku": {
+        "name": "Standard"
+      },
+      "tags": {
+        "myTag": "myTagValue"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "creationDate": "2018-04-24T16:30:55+00:00",
+          "endpoint": "https://contoso.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": {
+              "keyIdentifier": null,
+              "identityClientId": null
+            }
+          },
+          "disableLocalAuth": false,
+          "privateEndpointConnections": []
+        },
+        "systemData": {
+          "createdBy": "foo@contoso.com",
+          "createdByType": "User",
+          "createdAt": "2018-04-24T16:30:55+00:00",
+          "lastModifiedBy": "foo@contoso.com",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2018-04-24T16:30:55+00:00"
+        },
+        "sku": {
+          "name": "Standard"
+        },
+        "id": "/subscriptions/c80fb759-c965-4c6a-9110-9b2b2d038882/resourceGroups/myResourceGroup/providers/Microsoft.AppConfiguration/configurationStores/contoso",
+        "name": "contoso",
+        "location": "westus",
+        "tags": {
+          "myTag": "myTagValue"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "properties": {
+          "provisioningState": "Creating",
+          "creationDate": "2018-04-24T16:30:55+00:00",
+          "endpoint": "https://contoso.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": {
+              "keyIdentifier": null,
+              "identityClientId": null
+            }
+          },
+          "disableLocalAuth": false,
+          "privateEndpointConnections": []
```

Comment:
is the removal of publicNetworkAccess here intentional? your list api examples still have this property and so does your swagger

Reply:
@varunkumta this is intentional. `publicNetworkAccess` doesn't get returned until it is set. Since it isn't in the creation parameters in this example it wouldn't be returned. I noticed this inconsistency when I was editing the example.

It's in the list APIs because those demonstrate most properties having already been set.